extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -412,7 +413,7 @@ static public function setmainDomain ( $city_id, $domain = null, $reason = null
 
 		$retVal = WikiFactory::setVarByName("wgServer", $city_id, $domain, $reason);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "wgServer" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)